pkg/cmd/server/origin/master.go

```diff
@@ -607,7 +607,11 @@ func (c *MasterConfig) authorizationFilter(handler http.Handler) http.Handler {
 }
 
 // forbidden renders a simple forbidden error
-func forbidden(reason string, w http.ResponseWriter, req *http.Request) {
+func forbidden(reason, apiVersion string, w http.ResponseWriter, req *http.Request) {
+	if len(apiVersion) == 0 {
```

Comment:
Add a TODO here explaining the reason why this happens and the causes of it (what should be fixed, essentially)